operator/clusterutilities.go

```diff
@@ -866,3 +866,12 @@ func writeTablespaceJSON(w *bytes.Buffer, jsonFields interface{}) error {
 
 	return nil
 }
+
+// GetImagePrefix checks whether the CCPImagePrefix value given from the pgcluster
+// CRD has been set. If so, this value is returned, otherwise the default value is used.
+func GetImagePrefix(crdCCPImagePrefix string) string {
+	if crdCCPImagePrefix != "" {
+		return crdCCPImagePrefix
+	}
+	return Pgo.Cluster.CCPImagePrefix
+}
```

Comment:
Place this in alphabetical order in the file. (or as best as you can as the whole file is not in alpha order)

Reply:
sure thing. I put it directly above GetInstanceDeployments